### ISSUE TITLE
Add term transependymal edema

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10404]
**Preferred term label:**
Transependymal edema

**Synonyms**
interstitial cerebral edema, periventricular lucency, increased transependymal flow

**Definition (free text, please give PubMed ID)**
A type of cerebral edema that occurs with increased pressure within the cerebral ventricles. PMID: 34560294

**Parent term (use hpo.jax.org/app)**
Cerebral edema HP:0002181

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Communicating hydrocephalus (ICD10 G91.0)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw @azankl
The HPO does not encode pathophysiology or etiology. As far as I can see there is no characteristic phenotypic difference that would allow one to identify "Transependymal" as compared to other etiologies. Therefore, one would simply annotate "Cerebral edema".
